rep-0003.rst

```diff
@@ -273,7 +273,7 @@ Requirements:
 | Qt5     |     5.5.1     |     5.6.1      |    5.7.1     |    5.7.1       |   5.7.1   |
 +---------+---------------+----------------+--------------+----------------+-----------+
 
-" * " means that this is not the upstream version but a package distributed by OSRF or the community.
+" * " means that this is not the upstream version (available on the official Operating System repositories) but a package distributed by OSRF or the community (package built and distributed on custom repositories).
```

Comment:
+1
Thanks @mikaelarguedas for the update. Now I get what this sentence means, although I still had to think for awhile.

Maybe adding the concrete package names (`libopencv*`, `ros-lunar-opencv*` etc.) might help even more, but that would limit to OpenCV. I don't know.

Every time I explain about package distribution to people newer to opensource/Ubuntu I have to be verbal so I can't think of succinct wording here.